uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -49,44 +49,45 @@
 })
 
 M_PATH = 'uaclient.entitlements.livepatch.'  # mock path
+M_BASE_PATH = 'uaclient.entitlements.base.UAEntitlement.'
+
+
+@pytest.fixture
+def entitlement(tmpdir):
+    """
+    A pytest fixture to create a LivepatchEntitlement with some default config
+
+    (Uses the tmpdir fixture for the underlying config cache.)
+    """
+    cfg = config.UAConfig(cfg={'data_dir': tmpdir.strpath})
+    cfg.write_cache('machine-token', dict(LIVEPATCH_MACHINE_TOKEN))
+    cfg.write_cache('machine-access-esm', dict(LIVEPATCH_RESOURCE_ENTITLED))
```

Comment:
That didn't cause test failures because any tests that were specifically looking for config directives in the detailed machine-access-livepatch cache file had overwritten it for the specific test. As a result, I'm actually going to drop this 'setup' from the entitlement fixture, since it's currently unneeded/unused/overwritten.